tests/drivers/rtc/rtc_api_helpers/boards/frdm_k22f.conf

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright 2025 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_RTC_ALARM=y
+CONFIG_RTC_UPDATE=y
+CONFIG_RTC_CALIBRATION=y
```

Comment:
The helpers are purely functional, they are tested with emulated platforms, no need to add any real hardware to them, the RTC drivers are not used at all for the helpers and utils test suites.